cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4873,4 +4918,52 @@
       The maximum number of records that can be set in preview configuration.
     </description>
   </property>
+
+  <property>
+    <name>artifact.cache.bind.address</name>
+    <value>0.0.0.0</value>
+    <description>
+      The bind address for artifact service.
+    </description>
+  </property>
+
+  <property>
+    <name>artifact.cache.bind.port</name>
+    <value>11030</value>
+    <description>
+      The port to bind the artifact cache service to.
+    </description>
+  </property>
+
+  <property>
+    <name>artifact.cache.worker.threads</name>
+    <value>10</value>
+    <description>
+      The number of worker threads in the artifact cache service.
+    </description>
+  </property>
+
+  <property>
+    <name>artifact.cache.boss.threads</name>
+    <value>1</value>
+    <description>
+      The number of boss threads in the artifact cache service.
+    </description>
+  </property>
+
+  <property>
+    <name>artifact.cache.local.data.dir</name>
+    <value>/tmp</value>
```

Comment:
shall we have it under `${local.data.dir}` dir?

Reply:
set value to ${local.data.dir}/artifact.cache